.husky/pre-commit

```diff
@@ -1,4 +1,6 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
```

Comment:
@brena-odwyer thank you for your contribution, can you explain a little bit on this?
ling 1 and line 2?

Reply:
I saw those lines while researching how to implement husky on stack overflow. The first line is a hashbang, it is like a special comment in Unix-like operating systems that specifies the interpreter for the script. The second line, from what I understand,  is helping execute husky in different script locations. But I made a test on mine and tried commenting the second line (the same as the first) and it still worked the same way.

Reply:
Great, I will try to run it on my side.